packages/config/config/devices/0x0258/nas-pd01z.json

```diff
@@ -0,0 +1,173 @@
+		"1": {
+			"label": "Sensitivity Level",
+			"description": "Defines the sensitivity of PIR sensor. 8 highest sensitivity and 255 lowest sensitivity.",
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 12,
+			"unsigned": true
+		},
+		"2": {
+			"label": "On/Off Duration",
+			"description": "Determines how long the associated devices should stay on",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 30
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"description": "Basic Set Command will be sent when motion is triggered",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 99,
+			"unsigned": true
+		},
+		"4": {
+			"label": "PIR Detector",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 255
+				}
+			]
+		},
+		"5": {
+			"label": "Light Sensor Brightness Threshold",
+			"description": "The light sensor is active when the brightness is above this level",
+			"unit": "lux",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 1000,
+			"defaultValue": 100
+		},
+		"6": {
+			"label": "PIR Re-trigger Interval",
+			"description": "Must be less than the On/Off Duration",
+			"unit": "seconds",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 8,
+			"defaultValue": 8
+		},
+		"7": {
+			"label": "Light Sensor Polling Interval",
+			"description": "The Interval after which the Light Sensor Measure is polled.",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 60,
+			"maxValue": 36000,
+			"defaultValue": 180,
+			"unsigned": true
+		},
+		"8": {
+			"label": "Lux Level Function",
+			"description": "When enabled, send Basic Set if brighness falls below threshold.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"9": {
+			"label": "Lux Level Changed Report",
+			"description": "How much the Lux Level must change",
+			"unit": "lux",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 100,
+			"unsigned": true
+		},
+		"10": {
+			"label": "Temperature Differential Report Threshold",
+			"unit": "0.1 °C",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 5
+		},
+		"11": {
+			"label": "LED: Blink on Motion",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"12": {
+			"label": "Motion Event Report One Time Enable",
```

Comment:
I'm not very happy with this label and it's not clear what it does.
Got an idea for a better label?

Reply:
That's directly from the user manual.
My understanding is with it enabled, it will report motion only once during the On/off duration (parameter 2).
When disabled, it will continuosly report motion based on the redetection interval (parameter 6).

How about:
"Single/continuous motion event report enable"

Reply:
That makes much more sense. 👍